draft-schlesinger-cfrg-act.md

```diff
@@ -825,11 +840,61 @@ interoperability. The curve specification is included to prevent cross-curve
 attacks and ensure implementations using different curves cannot accidentally
 interact.
 
+### Sigma Proofs and Fiat-Shamir Transform {#sigma-and-fs}
```

Comment:
What's the goal of this text? If the goal is to present the APIs for and give a brief overview of  {{SIGMA}} and {{FIAT-SHAMIR}}, then it should probably go before the ACT protocol definition.